Push Migration Plugin side (#947)
salesforcecli/plugin-packaging: messages/package_pushupgrade_schedule.md

Comma-separated list of subscriber org IDs that need the package upgrade.

> # flags.migrate-to-2gp.summary

Author: Messages being reviewed by Penelope

> Indicates the push upgrade request is a migration from 1GP to 2GP.

Reviewer: ```suggestion
Migrate the specified 1GP package to a 2GP package.
```

Reviewer: Okay?  The [CLI style guidelines](https://github.com/salesforcecli/cli/wiki/Write-Useful-Messages#flag-summary) say that the summary for a Boolean flag must start with an imperative word.

Reviewer: FYI @penelopy
